source/designers-guide/css-and-js-files-usage/index.md

```diff
@@ -1,16 +1,20 @@
 ---
 layout: default
-title: Using CSS and JavaScript files in themes.
+title: Using CSS and JavaScript in themes.
 github_link: designers-guide/css-and-js-files-usage/index.md
 indexed: true
 group: Frontend Guides
 subgroup: Developing Themes
-menu_title: Using CSS and JavaScript files
+menu_title: Using CSS and JavaScript
 menu_order: 50
 ---
 
-This quick tip shows off the best way on how to use CSS and JavaScript files for your custom themes, in order to enable them to be automatically compressed by the Shopware theme compiler. To use this feature you have to place your CSS and JavaScript files inside your theme directory under the subdirectories `frontend/_public`. This would be an example directory structure:
+<div class="toc-list"></div>
 
+## The theme compiler
+Shopware uses a compiler to minify and concatenate resource files like CSS and JavaScript to single compressed files. This reduces file size and the amount of server requests. It is highly recommended to also adding your files to the compiler. The generated files are added to the template automatically, so you don't have to worry about including your files at all.
+
+Before you add your files to the theme compiler you have to place them in the correct directory structure `frontend/_public/` inside your custom theme. Here is an example directory structure:
```

Comment:
I would get rid of the example `frontend/_public/` here as you already show a tree below.